package.json

```diff
@@ -24,6 +25,7 @@
     "web"
   ],
   "dependencies": {
+    "bower": "^1.4.1",
```

Comment:
Yeah unfortunately this won't work, as Heroku does not have bower (or grunt) installed and cannot install it by default since Heroku uses the `production` flag when running `npm install` (meaning no dev depencies are installed). 

We can try one of two things here:
- Include bower and grunt-cli as devDependency and change the build config variable `NPM_CONFIG_PRODUCTION` from `true` to `false`; the cost to this would be installing all dev dependencies.
- Leave bower and add grunt-cli as local deps at the cost of including them when someone runs `npm install`, nothing else needs to be changed. 

Unless you can think of another way around this, those are our options.